app/src/main/AndroidManifest.xml

```diff
@@ -4,19 +4,32 @@
 
     <application
         android:allowBackup="true"
+        android:fullBackupContent="@xml/backup_descriptor"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme"
-        android:fullBackupContent="@xml/backup_descriptor">
+        android:theme="@style/AppTheme">
+
+        <!-- Iskomentirana glavna aktivnost kako bi se na pocetnom ekranu pokazivao profil korisnika, samo privremeno rjesenje -->
+        <!--
         <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        -->
+
+        <activity android:name=".ProfileActivity" />
+        <activity android:name=".EditProfileActivity">
```

Comment:
Before commit this should be returned to it's original state. Otherwise it's OK.